proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,536 @@
+4. Sie nimmt den Jahresbericht des Vorstandes und den Bericht der
+   Rechnungsprüfer entgegen und erteilt dem Vorstand Entlastung.
+5. Sie beschließt über Satzungsänderungen und die Auflösung des Vereins.
+6. Ihr sind insbesondere die Jahresrechnung und der Jahresbericht zur
+   Beschlussfassung über die Genehmigung und die Entlastung des Vorstandes
+   schriftlich vorzulegen. Sie beruft einen Rechnungsprüfer, die weder dem
+   Vorstand noch einem vom Vorstand berufenen Gremium angehören und auch nicht
+   beim Berein angestellt ist, um die Buchführung einschließlich Jahresabschluss
+   zu prüfen und über das Ergebnis vor der Mitgliederversammlung zu berichten.
+   Dr Rechnungsprüfer hat Zugang zu allen Buchungs- und Rechnungsunterlagen des
+   Vereins.
+7. Sie beschließt über Käufe, Verkäufe, Investitionen und die Aufnahme von
+   Krediten.
+
+## 8. Der Vorstand
+
+1. Der Vorstand besteht aus dem Vorsitzenden, dem zweiten Vorsitzenden und dem
+   Schatzmeister die für eine Amtszeit von 3 Jahren gewählt werden.
+
+   TODO: Definieren Sie die Verantwortlichkeiten der Rollen.
+
+2. Die Aufgaben des Vorstands und der Ablauf der Sitzungen sind wird durch eine
+   Geschäftsordnung des Vorstands geregelt.
+3. Der Vorstand beschließt über alle Vereinsangelegenheiten, soweit sie nicht
+   eines Beschlusses der Mitgliederversammlung bedürfen. Er führt die Beschlüsse
+   der Mitgliederversammlung aus.
+4. Jedes Vorstandsmitglied hat das Recht, den Verien zu vertreten.
+5. Im Falle des Ausscheidens eines Vorstandsmitglieds während der Amtszeit
+   bestimmt der Vorstand auf basis einer Internetwahl ein kommissarisches
+   Vorstandsmitglied. Das kommissarische Vorstandsmitglied bleibt bis zur
+   nächsten Mitgliederversammlung im Amt. Über eine endgültige Nachfolge im
+   Vorstand entscheidet die Mitgliederversammlung.
+6. Im Falle des Ausschlusses eines Vorstandsmitglieds gemäß §4 (5) wird dieses
+   durch ein kommissarisches Vorstandsmitglied ersetzt.
+7. Der Vorstand kann zur Führung der laufenden Geschäfte einen hauptamtlichen
+   Geschäftsführer bestellen.
+8. Der Geschäftsführer nimmt an den Mitgliederversammlungen und
+   Vorstandssitzungen teil.
+9. Der Vorstand kann die Satzung aus formalen Gründen ändern, wenn von den
+   Behörden verlangt wird.
+
+## 9. Geschäftsordnung
+
+Die Geschäftsordnung regelt über den Rahmen der Satzung hinausgehende Details
+der Vereinsarbeit. Der Vorstand setzt die Geschäftsordnung nach der Zustimmung
+der Kernmitglieder in einer Internetabstimmung in Kraft.
+
+## 10. Protokolle
+
+Alle Beschlüsse des Vorstandes und der Mitgliederversammlung werden schriftlich
+in deutscher Sprache dokumentiert und stehen den Mitgliedern zur Einsichtnahme
+und zur Prüfung der Beschlüsse durch das Registergericht zur Verfügung.
+
+## 11. Internetabstimmung
+
+Die Vereinigung kann Abstimmungen über das Internet durchführen. Eine
+Internetabstimmung ist gültig nach einer Ankündigung per E-Mail oder Messenger
```

Comment:
def "messenger"? Zulip? [facebook messenger](https://en.wikipedia.org/wiki/Messenger_(software))?

Reply:
sounds good. though I'm not sure if "obligatorisch" is the right word...

Reply:
This should mean that all core members have to join the core stream. 
Do you agree with that?

Reply:
Can we add a clause like "The general assembly decides about the internet communication system used for online voting"